### PR TITLE
Shitmed Update 1 - Forma de Goob

### PR DESCRIPTION
**Pull request by gluesniffler**
## About the PR
I fixed some shitcode. Moved some things around to the _Shitmed namespace.

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Mocho, Deltanedas
- tweak: Cyborg arms/legs can now be used as suplementary body parts.
- tweak: Entities now properly take asphyxiation damage after losing their brain.
- tweak: Entities now perish after 60 seconds of losing their heart and/or brain.
- fix: Items not being removed from their respective slots if the parts were gibbed rather than dropped.
- fix: TORSOS NO LONGER DISAPPEAR, PRAISE BE THE LORD.
- fix: Ghetto surgical tools are no longer faster than the advanced ones.
- fix: Fixed Cavity Insertions not working due to shitcode.
- fix: You can no longer reuse your organs over and over to abuse transplant healing.
- fix: Organs are now properly "spent" once they've been used in a transplant surgery.

## Release Notes

- **New Features**
	- Introduced new components for managing organ interactions, including `HeartSystem` and `DelayedDeathComponent`.
	- Added new status effects and refined existing ones to enhance gameplay mechanics.

- **Improvements**
	- Enhanced body and part gibbing logic for better control and event handling.
	- Updated various tools and weapons with new sound effects and attributes.

- **Bug Fixes**
	- Improved error handling and control flow in surgery and organ management systems.

- **Documentation**
	- Updated comments and documentation for clarity, reflecting the new branding and context changes from "GoobStation" to "Shitmed."

### CODE REVIEW

**Comment by whateverusername0** [comment]
but what will happen with heartless/brainless dudes on a stasis bed? @gluesniffler

File: Content.Server/_Shitmed/DelayedDeath/DelayedDeathSystem.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            component.DeathTimer += frameTime;
+
+            if (component.DeathTimer >= component.DeathTime && !_mobState.IsDead(ent))
+            {
+                var damage = new DamageSpecifier(_prototypes.Index<DamageTypePrototype>("Bloodloss"), 150);
+                _damageable.TryChangeDamage(ent, damage, partMultiplier: 0f);
+            }
+        }
+    }
```
[the remark this answers is not in the document]
Comment (gluesniffler) [reply Content.Server/_Shitmed/DelayedDeath/DelayedDeathSystem.cs:27]: I want to kill this system pretty badly anyway so im not gonna put more effort than needed.

File: Content.Server/_Shitmed/Medical/Surgery/SurgerySystem.cs [hunk 2]
```diff
@@ -129,33 +131,21 @@ private void OnToolAfterInteract(Entity<SurgeryToolComponent> ent, ref AfterInte
...
-            return;
-
         if (ent.Comp.DamageType == "Rot")
             _rot.ReduceAccumulator(args.Body, TimeSpan.FromSeconds(2147483648)); // BEHOLD, SHITCODE THAT I JUST COPY PASTED. I'll redo it at some point, pinky swear :)
         else if (ent.Comp.DamageType == "Eye"
-            && TryComp(args.Body, out BlindableComponent? blindComp)
+            && TryComp(ent, out BlindableComponent? blindComp)
             && blindComp.EyeDamage > 0)
             _blindableSystem.AdjustEyeDamage((args.Body, blindComp), -blindComp!.EyeDamage);
     }
```
[the remark this answers is not in the document]
Comment (gluesniffler) [reply Content.Server/_Shitmed/Medical/Surgery/SurgerySystem.cs:151]: i love shitcode, and this is a far funnier reminder that i need to do something else for heart surgeries anyway

**Comment by gluesniffler** [comment]
@Aidenkrz goobers are dying, billions of torsos are becoming invisible, only you can save them, merge button man

**Comment by Aidenkrz** [comment]
> @Aidenkrz goobers are dying, billions of torsos are becoming invisible, only you can save them, merge button man

resolve conflicts :trollface:

**Pull request merged**